taiga_halo2/examples/sudoku/app/valid_sudoku/circuit.rs

```diff
@@ -163,8 +163,18 @@ impl plonk::Circuit<pallas::Base> for SudokuCircuit {
         // The length of u is 41 bytes, or 328 bits, since we are allocating 4 bits
         // per the first 40 integers and let the last sudoku digit to take an entire byte.
         // We still need to add 184 bits (i.e. 23 bytes) to reach 2*256=512 bits in total.
-        let u2 = [u, vec![0; 23]].concat();
-        let u_first: [u8; 32] = u2[0..32].try_into().unwrap();
+        let mut u2 = [0u8; 64];
+        let mut i = 0;
+        let mut j = 0;
+        while j != u.len() {
```

Comment:
Could you write a comment about this change?

Reply:
Done.